testnet/stacks-node/src/nakamoto_node/sign_coordinator.rs

```diff
@@ -748,6 +773,18 @@ impl SignCoordinator {
                 return Ok(stored_block.header.signer_signature);
             }
 
+            if new_burn_tip_ts.is_none() {
+                if Self::check_burn_tip_changed(&sortdb, &burn_tip.consensus_hash) {
+                    new_burn_tip_ts = Some(Instant::now());
+                }
+            }
+            if let Some(ref new_burn_tip_ts) = new_burn_tip_ts.as_ref() {
+                if new_burn_tip_ts.elapsed() >= self.config.miner.wait_on_interim_blocks {
```

Comment:
Thinking about this some more since our last discussion... what does the miner gain from waiting for the block approval in this case?
1. It knows it cannot mine any blocks after this one in this tenure.
2. It knows that it will eventually receive the signed block via the signers.
3. It knows a new tenure has started, so it is in its best interest to submit a block commit quickly.
4. If it won the new tenure, it is in its best interest to start mining blocks, to ensure it gets its coinbase before the next tenure starts.

Based on that, I propose to drop this time, and just immediately return when it sees the new tenure.

Reply:
Alright, I'll remove.  IIRC the rationale was that we wanted to give the signer a chance to "finish up" any in-flight blocks, but that should be irrelevant now that signers can take care of processing block signing and broadcasting themselves.